cohttp-curl-async/src/dune

```
@@ -0,0 +1,11 @@
+(library
+ (name cohttp_curl_async)
+ (public_name cohttp-curl-async)
```

```suggestion
 (public_name cohttp-curl-async)
 (package cohttp-curl-async)
```
Why is package not needed here? Is it because the public name and the package name coincide?